torch/onnx/symbolic_opset9.py

```diff
@@ -1970,3 +1966,33 @@ def gelu(g, self):
     erf = g.op('Erf', div(g, self, torch.tensor(_sqrt2)))
     erf_plusone = add(g, erf, g.op('Constant', value_t=torch.tensor(1, dtype=torch.float)))
     return mul(g, mul(g, self, erf_plusone), g.op('Constant', value_t=torch.tensor(0.5, dtype=torch.float)))
+
+
+@parse_args('v', 'i', 'v', 'v', 'f', 'i')
+def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
+    input_sizes = input.type().sizes()
+    shape = [0, num_groups, -1]
```

Comment:
what's the meaning of 0?

This reshape doesn't look good to me.

Reply:
onnx::reshape docs mention that using 0 in shape tensor let's you to infer the shape from input.
https://github.com/onnx/onnx/blob/master/docs/Operators.md#reshape
(actual dimension value would be unchanged). We are using this to avoid statically accessing the shape.